slides/intro.md

```diff
@@ -89,7 +77,7 @@ Direct all complaints concerning grades to her
 * UI components using React
 * Means of composition
 * Testing, testing, testing
---
+  --
```

Comment:
This slide looks a bit odd now:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/18231426/153058287-a87ef524-20a5-4c80-bf01-b051888cfe71.png">

Let's split it into 2?